<?xml version="1.0" encoding="UTF-8"?>
<root BTCPP_format="4">
  <BehaviorTree ID="main">
    <Sequence>
      <MoveToPose posX="1.5"
                  posY="-3"
                  angle="135"
                  velX="0.5"
                  velY="-0.5"
                  omega="0"/>
      <KeepRunningUntilFailure>
        <SaySomething message="doing stuff"/>
      </KeepRunningUntilFailure>
    </Sequence>
  </BehaviorTree>

  <!-- Description of Node Models (used by Groot) -->
  <TreeNodesModel>
    <Action ID="MoveToPose"
            editable="true">
      <input_port name="posX"/>
      <input_port name="posY"/>
      <input_port name="angle"/>
      <input_port name="velX"/>
      <input_port name="velY"/>
      <input_port name="omega"/>
    </Action>
    <Action ID="SaySomething"
            editable="true">
      <input_port name="message"/>
    </Action>
  </TreeNodesModel>

</root>
